November newsletter (#459)
readthedocs/ethicalads.io: content/posts/newsletter-november-2024.md

  features we built for our ad client. We're continuing to iterate here,
  although we've scaled back rotations somewhat.
  We also building some reporting features around this.

You can always see our latest server updates in our
[ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html)
and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## October advertising stats

[comment]: https://server.ethicalads.io/publisher/all/report/?start_date=2024-10-01&end_date=2024-10-31

Last month:

* We generated **$41,677.24** for our publishers and they'll receive this payout next week.
* We had **181 publishers** on our network with at least one paid ad impression.
* We had **26,279,987** paid ad views across the world.
  This is the highest it's ever been and our new run of network options
  are allowing us to sell a larger percentage of our traffic especially in regions
  that were historically undersold.
  We also have some very broad campaigns running.

We are seeing some softness in our core markets
after a few mid-sized advertisers put campaigns on hold

> and a larger one who wasn't expected to renew pulled back.

Reviewer: ```suggestion
and a larger one pulled back.
```
Not sure what this is saying?

We are running a November discount for new advertisers
where we're offering the discounts we [normally]({filename}../pages/advertisers-pricing.md)
give at $3k and $25k at lower thresholds ($1k and $5k).
If you're interested in running a campaign, [get in touch]({filename}../pages/advertisers.md#inbound-form)!

## Upcoming features

The major features in our upcoming roadmap for November:

* We are building a bulk importer to handle uploading ads in bulk.
  A couple advertisers have asked for this to speed up the process
  when running multiple campaigns.

> * We should be releasing some new advertiser features to have lower privilege accounts
>   that can just see report data but not make any changes to ads themselves.

Reviewer: ```suggestion
* We'll be releasing some new advertiser features to have read-only accounts
  that can just see report data but not make any changes to ads themselves.
```

Should we make these on publishers as well, while we're at it?
